fluid/DeepQNetwork/DuelingDQN_agent.py

```diff
@@ -1,10 +1,10 @@
 #-*- coding: utf-8 -*-
 
+import math
+import numpy as np
 import paddle.fluid as fluid
 from paddle.fluid.param_attr import ParamAttr
-import numpy as np
 from tqdm import tqdm
-import math
 from utils import fluid_flatten
```

Comment:
fluid support `flatten`, too.
Please remove file `utils` and use fluid general API.